toolz/itertoolz.py

```diff
@@ -26,7 +26,7 @@ def remove(predicate, seq):
     return filterfalse(predicate, seq)
 
 
-def accumulate(binop, seq):
+def accumulate(binop, seq, start=no_default):
```

Comment:
Yeah, I regret using `init`.  I would use `initial` in `accumulate` and leave `reduceby` as is for now.  The `reduceby` change would have to go through a deprecation cycle (or just support both.)